Dockerfile

```diff
@@ -37,12 +37,10 @@ ENV GOPATH /go
 ENV PATH /go/bin:$PATH
 ENV PATH /usr/local/go/bin:$PATH
 
-RUN mkdir -p /go/src/github.com/alecthomas && \
-    cd /go/src/github.com/alecthomas && \
-    git clone https://github.com/alecthomas/gometalinter.git --branch=v1.2.1 && \
-    go install -v github.com/alecthomas/gometalinter && \
-    gometalinter --install && \
-    go get -u golang.org/x/tools/cmd/goimports && \
+RUN mkdir -p /go/src/github.com/golangci/ && \
+    cd /go/src/github.com/golangci && \
+    git clone https://github.com/golangci/golangci-lint.git --branch=v1.17.1 && \
```

Comment:
Let's move the version number into a constant like we do for other dependencies for easier maintenance.